src/runtime/virtcontainers/fs_share_linux.go

```diff
@@ -542,6 +543,57 @@ func (f *FilesystemShare) shareRootFilesystemWithVirtualVolume(ctx context.Conte
 	}, nil
 }
 
+func (f *FilesystemShare) shareRootFilesystemWithErofs(ctx context.Context, c *Container) (*SharedFile, error) {
+	guestPath := filepath.Join("/run/kata-containers/", c.id, c.rootfsSuffix)
```

Comment:
Optional suggestion - should we extract `"/run/kata-containers/"` to a constant somewhere?